### PR TITLE
fix: add pod / ns env to echo server

### PR DESCRIPTION
**Pull request by Xunzhuo**
Signed-off-by: bitliu <bitliu@tencent.com>

### CODE REVIEW

File: examples/kubernetes/quickstart.yaml [hunk 1]
```diff
@@ -59,6 +59,15 @@ spec:
           name: backend
           ports:
             - containerPort: 3000
+          env:
```
Comment (arkodg) [inline examples/kubernetes/quickstart.yaml:62]: can you share the impact - before/after ?

Comment (arkodg) [reply examples/kubernetes/quickstart.yaml:62]: nvm just saw this https://github.com/envoyproxy/gateway/issues/591

Comment (arkodg) [reply examples/kubernetes/quickstart.yaml:62]: can you please attach a log :)

Comment (Xunzhuo) [reply examples/kubernetes/quickstart.yaml:62]: before:
image

after:

image

**Review by arkodg — APPROVED** [review]
🚀

**Pull request merged**